tools-local/unix/acquire-tool.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+# Downloads the declared version of the specified tool.
+# Download URL and package name corresponding to the tool is read from the .toolversions file.
+# Arguments:
+#   1. Name of the tool
```

Comment:
Add usage for all scripts.

---
In reply to: [107990675](https://github.com/ravimeda/corefx/pull/2#discussion_r107990675) [](ancestors = 107990675)